### PR TITLE
New Package: audiofile v0.3.6

### PR DESCRIPTION
**Pull request by the-maldridge**

### CODE REVIEW

File: srcpkgs/audiofile/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version=0.3.6
+revision=1
+wrksrc=$pkgname-$pkgname-$version
+build_style=gnu-configure
+hostmakedepends="automake libtool asciidoc pkg-config"
+makedepends="gtk+-devel alsa-lib-devel"
+short_desc="C library for reading and writing audio files"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="GPL-2"
+homepage="audiofile.68k.org"
```
Comment (dominikh) [inline srcpkgs/audiofile/template]: URL needs a protocol.

File: srcpkgs/audiofile/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+disable_parallel_build=1
+distfiles="http://github.com/mpruett/audiofile/archive/audiofile-$version.tar.gz"
+checksum=52125fee6c7454d743acdc27ebda194c6b5c7b9111426c7d5fdea0754cd366cc
+
+pre_configure() {
+	./autogen.sh
+}
+
+audiofile-devel_package() {
```
Comment (dominikh) [inline srcpkgs/audiofile/template]: Please fix the indentation

File: srcpkgs/audiofile/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+short_desc+=" - development files"
+depends="audiofile>=${version}_${revision}"
+pkg_install() {
+vmove usr/lib/*.so
+vmove usr/lib/*.la
+vmove usr/lib/*.a
+vmove usr/lib/pkgconfig/
+vmove usr/include/*.h
```
Comment (dominikh) [inline srcpkgs/audiofile/template]: `vmove usr/include`

File: srcpkgs/audiofile/template [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+vmove usr/share/man/man3*
```
Comment (dominikh) [inline srcpkgs/audiofile/template]: `vmove usr/share/man/man3`

File: srcpkgs/audiofile/template [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+pkgname=audiofile
+version=0.3.6
+revision=1
+wrksrc=$pkgname-$pkgname-$version
+build_style=gnu-configure
+hostmakedepends="automake libtool asciidoc pkg-config"
+makedepends="gtk+-devel alsa-lib-devel"
+short_desc="C library for reading and writing audio files"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="GPL-2"
```
Comment (dominikh) [inline srcpkgs/audiofile/template]: It uses the LGPL-2.1: https://github.com/mpruett/audiofile/commit/3b4f40088c1cf6f36fd8c6d517bb854ff1a8d9bd

File: srcpkgs/audiofile/template [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+wrksrc=$pkgname-$pkgname-$version
+build_style=gnu-configure
+hostmakedepends="automake libtool asciidoc pkg-config"
+makedepends="gtk+-devel alsa-lib-devel"
+short_desc="C library for reading and writing audio files"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="LGPL-2.1"
+homepage="http://audiofile.68k.org"
+disable_parallel_build=1
+shlib_requires="libaudiofile.so.1"
```
Comment (xtraeme) [inline srcpkgs/audiofile/template]: I really doubt this is necessary, can you explain?

File: srcpkgs/audiofile/template [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+pre_configure() {
+	./autogen.sh
+}
+
+audiofile-devel_package() {
+	short_desc+=" - development files"
+	depends="audiofile>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/lib/*.so
+		vmove usr/lib/*.la
```
Comment (xtraeme) [inline srcpkgs/audiofile/template]: Libtool archives are removed automagically by xbps-src, why do you need this?

File: srcpkgs/audiofile/template [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+		vmove usr/lib/*.a
+		vmove usr/lib/pkgconfig/
+		vmove usr/include/
+		vmove usr/share/man/man3/
+	}
+}
+
+post_install() {
+	vlicense COPYING
```
Comment (xtraeme) [inline srcpkgs/audiofile/template]: `GPL` licenses do not need to be installed.

**Comment by dominikh** [comment]
Also missing the common/shlibs addition.

**Pull request closed without merging**